src/components/dialogs/load-creation-dialog.js

```diff
@@ -15,17 +21,29 @@ import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
 import { useSnackbar } from 'notistack';
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import { FormattedMessage, useIntl } from 'react-intl';
 import { useParams } from 'react-router-dom';
 import {
     displayErrorMessageWithSnackbar,
     useIntlRef,
 } from '../../utils/messages';
-import { createLoad } from '../../utils/rest-api';
+import {
+    createLoad,
+    fetchEquipmentsInfos,
+    fetchLoadInfos,
+} from '../../utils/rest-api';
 import TextFieldWithAdornment from '../util/text-field-with-adornment';
 import { makeErrorHelper, validateField } from '../util/validation-functions';
 import ConnectivityEdition from './connectivity-edition';
+import { PARAM_USE_NAME } from '../../utils/config-params';
+import FindInPageIcon from '@material-ui/icons/FindInPage';
```

Comment:
PO asked for ContentPasteSearch icon (https://mui.com/components/material-icons/?query=content&selected=ContentPasteSearch) is it a change in the US ?

Reply:
icon required an upgrade on mui so we decided to go with another one